crates/sui-graphql-rpc/schema/draft_target_schema.graphql

```diff
@@ -300,13 +321,21 @@ input EventFilter {
   afterCheckpoint: Int
   beforeCheckpoint: Int
 
-  # Cascading (module requires package)
-  emittingPackage: SuiAddress
+  # Events emitted by a particular module. An event is emitted by a
+  # particular module if some function in the module is called by a
+  # PTB and emits an event.
+  #
+  # Modules can be filtered by their package, or package::module.
```

Comment:
@wlmyng , heads up that I extended the plan to collapse "cascading" filters from types to modules.